src/libsyntax/lib.rs

```diff
@@ -105,6 +105,7 @@ pub mod syntax {
     pub use ext;
     pub use parse;
     pub use ast;
+    pub use tokenstream;
```

Comment:
I think that is not required anymore, as you don't use the `quote!` anymore.

Reply:
Yep

Reply:
Do you still need to remove this?